src/ducks/apps/components/Sections/components/Filters.jsx

```diff
@@ -10,18 +10,17 @@ import SettingIcon from 'cozy-ui/transpiled/react/Icons/Setting'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
-import { useEffect } from 'react'
 
 const { BarRight } = cozy.bar
 
 const Filters = ({ filter, onFilterChange }) => {
   const anchorRef = useRef()
   const [menuDisplayed, setMenuDisplayed] = useState(false)
-  const [appUnderMaintenance, setAppUnderMaintenance] = useState(true)
 
-  useEffect(() => {
-    setAppUnderMaintenance(filter.underMaintenance == undefined)
-  }, [filter.underMaintenance])
+  const appUnderMaintenance = useMemo(
+    () => filter.underMaintenance == undefined,
```

Comment:
Also this seems weird that `underMaintenance == undefined` means that `appUnderMaintenance = true`. I'm pretty sure renaming those variable would ease the comprehension